### PR TITLE
Feature/avo 1833

### PR DESCRIPTION
**Pull request by ian-emsens-shd**
https://meemoo.atlassian.net/browse/AVO-1833

### CODE REVIEW

**Review by bertyhell — APPROVED** [review]

File: src/settings/components/LinkedAccounts.tsx [hunk 1]
```diff
@@ -205,10 +212,8 @@ const LinkedAccounts: FunctionComponent<AccountProps> = ({ location, user }) =>
 										'settings/components/account___koppel-je-account-met-andere-platformen'
 									)}
 								>
-									{/* Disabled for pupils due to https://meemoo.atlassian.net/browse/AVO-2062 */}
-									<div>
-										{!isPupil && renderIdpLinkControls('VLAAMSEOVERHEID')}
-									</div>
+									{/* Previously disabled for pupils due to https://meemoo.atlassian.net/browse/AVO-2062 */}
```
Comment (bertyhell) [inline src/settings/components/LinkedAccounts.tsx:215]: do we still need this?

File: src/settings/components/LinkedAccounts.tsx [hunk 2]
```diff
@@ -205,10 +212,8 @@ const LinkedAccounts: FunctionComponent<AccountProps> = ({ location, user }) =>
...
+									<div>{renderIdpLinkControls('VLAAMSEOVERHEID')}</div>
```
Comment (bertyhell) [inline src/settings/components/LinkedAccounts.tsx:216]: button is not the same size on mobile:
image

**Review by bertyhell — CHANGES_REQUESTED** [review]

File: src/settings/components/LinkedAccounts.tsx (hunk 1 above)
Comment (ian-emsens-shd) [reply src/settings/components/LinkedAccounts.tsx:215]: I'd prefer to keep the record here

**Comment by bertyhell** [comment]
The link screen is quite confusing now.
image


Would we change it to this design instead?
image

**Pull request merged**